src/main/java/yesman/epicfight/api/client/event/types/BuildCameraTransform.java

```diff
@@ -23,9 +24,24 @@ public float getPartialTick() {
     }
 
     public static final class Pre extends BuildCameraTransform implements CancelableEvent {
+        private boolean cancelVanillaCameraSetup = false;
+
         public Pre(EpicFightCameraAPI cameraApi, Camera camera, float partialTick) {
             super(cameraApi, camera, partialTick);
         }
+
+        /// Set true if you want to disable the vanilla camera setup process
+        /// This means, you have modified the camera transform through this event
+        /// and don't want it to be overwrite by vanilla camera setups.
+        ///
+        /// @see MixinCamera#epicfight$setup
```

Comment:
```suggestion
		/// Set to `true` to disable the vanilla camera setup process.
		/// Indicates that the camera transform has been modified through this event
		/// and should not be overwritten by vanilla camera setups.
		///
		/// @see MixinCamera#epicfight$setup
```

From [this comment](https://github.com/Epic-Fight/epicfight/pull/2276#discussion_r2580242713)